lit/docs/jobs.lit

```diff
@@ -67,18 +67,29 @@ configured job consists of the following fields:
     \italic{Optional.} Keep logs for the last specified number of builds.
   }{job-build-log-retention-builds}
 
+  \define-attribute{minimum_succeeded_builds: number}{
+    \italic{Optional.} Keep at lease specified succeeded builds. For example,
+    if specifies \reference{job-build-log-retention-builds} to 5, and this
+    attribute to 1, say a job has a build history: 7(f), 6(f), 5(f), 4(f),
```

Comment:
```suggestion
    attribute to 1, say a job has the following build history: 
    7(f), 6(f), 5(f), 4(f), 3(f), 2(f), 1(s),
```

Reply:
(btw I know this line break won't render this way in the docs, but it makes it easier to reason about in Markdown)